labonneboite/tests/selenium/test_results.py

```diff
@@ -9,7 +9,12 @@ def test_toggle_office_details(self):
         """
         Tests the toggle mechanism of the `results` page.
         """
-        url = self.url_for('search.results', city='metz', zipcode='57000', occupation='comptabilite')
+        url = self.url_for(
+            'search.results',
+            city='metz',
+            zipcode='57000',
+            occupation='comptabilite'
```

Comment:
vieux débat de la comma en fin du dernier élément de liste :-)

je pousse pas

Reply:
TIL why trailing commas are so useful: https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list.
Je change !